binderhub/static/js/App.jsx

```diff
@@ -0,0 +1,111 @@
+import { LoadingPage } from "./pages/LoadingPage.jsx";
+import { Route, Router, Switch } from "wouter";
+import "bootstrap/js/dist/dropdown.js";
+
+import "./index.scss";
+import "@fontsource/clear-sans/100.css";
+import "@fontsource/clear-sans/300.css";
+import "@fontsource/clear-sans/400.css";
+import { HomePage } from "./pages/HomePage.jsx";
+import { AboutPage } from "./pages/AboutPage.jsx";
+import { NotFoundPage } from "./pages/NotFoundPage.jsx";
+
+export const PAGE_CONFIG = window.pageConfig;
+
+/**
+ * @typedef {object} RepoConfig
```

Comment:
@manics soooo you can run `npx tsc --noEmit` locally to have typescript validate these (it'll read JS, doesn't require typescript). However, the validation doesn't actually pass, mostly because our spec isn't fully complete. Given this PR has already taken so long, I want to not block this PR on that. Does that seem ok?

Reply:
Sure, it's not a blocker. You're the only one who knows about it though, so can you make a note of it somewhere, either an issue, in the docs, or somewhere else?

Reply:
@manics opened https://github.com/jupyterhub/binderhub/issues/1898!